telegram/_bot.py

```diff
@@ -1100,6 +1104,13 @@ async def send_audio(
                     File paths as input is also accepted for bots *not* running in
                     :paramref:`~telegram.Bot.local_mode`.
 
+                .. deprecated:: NEXT.VERSION
+                   Bot API 6.6 renamed this argument to :paramref:`thumbnail`.
```

Comment:
```suggestion
                   As of Bot API 6.6 this argument is deprecated in favor of :paramref:`thumbnail`.
```

In the current phrasing one might wonder why the `thumb` parameter still exists :) btw, you can also use a substitution for this if you like to :)

Reply:
Done (couldn't include reference in the substitution though)